<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleWidget</class>
 <widget class="QWidget" name="RuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RuleWidget</string>
  </property>
  <widget class="QLabel" name="ruleTitle">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>35</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>遊戲規則</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ruleText">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>130</y>
     <width>930</width>
     <height>475</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>620</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  返回</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
  </widget>
  <widget class="QLabel" name="soundPreview">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>272</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>音效預覽</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="corrSoundPreview">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>377</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>甜美的恭喜音效</string>
   </property>
  </widget>
  <widget class="QPushButton" name="incorrSoundPreview">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>460</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>甜美的鼓勵音效</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>280</y>
     <width>20</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
